### DISCUSSION TITLE
The latest update seems to not work for me

### DISCUSSION THREAD

**Discussion by exeqtwasnotfree** [atrocore/atrocore#728] [General]
Hello
After this weekends update from version:
Core 1.5.63 to 1.5.64
Pim 1.8.11 to 1.8.12
DAM 1.5.9 to 1.5.10
Import 1.4.59 to 1.4.60 
Export 1.6.46 to 1.6.47

I get the "Error 500" showing on both my sites, following error shows both in apache logs, and cron daemon email me the same error:

`PHP Fatal error:  Declaration of Pim\Repositories\ProductAttributeValue::getChildrenArray(string $parentId, bool $withChildrenCount = true, ?int $offset = null, $maxSize = null): array must be compatible with Espo\Core\Templates\Repositories\Hierarchy::getChildrenArray(string $parentId, bool $withChildrenCount = true, ?int $offset = null, $maxSize = null, $selectParams = null): array in /var/www/pim2/vendor/atrocore/pim/app/Repositories/ProductAttributeValue.php on line 128`

Where can i begin to troubleshoot this?
Can i reverse the update, or restore from backup?

**Comment by rratsun** [comment]
Please, update PIM to 1.8.13. We have fixed that bug there.

**Reply by rratsun** [reply]
Sorry, to 1.8.14. One more bug was found.

**Reply by exeqtwasnotfree** [reply]
Thank you for fast reply and the solution works, we are up and running again, thank you!
